src/utils/internal.ts

```diff
@@ -1891,3 +1892,28 @@ export function areSameClaims(
 
   return ClaimType[type] === claim.type;
 }
+
+/**
+ * @hidden
+ */
+export function assertCaAssetValid(id: string): string {
```

Comment:
this isn't a simple assert, but it transforms the value, so shouldn't have the `assert` prefix.

Unless I am  missing something we should not insert the `-`. An actual UUID doesn't seem good for privacy. UUIDs generated on one computer can get grouped together with v4, v7 has time sorting, etc.